### ISSUE TITLE
Use readthedocs or similar, build API docs

### ISSUE THREAD

**Issue by elcorto** [psteinb/c2st#16]

**Comment by elcorto** [comment]
Without building the docs using sphinx or other tools, formatting errors won't surface.

**Comment by psteinb** [comment]
Yes, please. Let's do this. I simply didn't have the capacity to set this up until now. Feel free to help with this.

**Comment by elcorto** [comment]
Same here :) This is merely a mental note that we should strive to do that. 

A minimal version would be to at least build a gh-pages version of the README + API docs, [as I do in psweep](https://github.com/elcorto/psweep/blob/master/.github/workflows/gh-pages.yml). When building API docs, we'd have at least a "syntax check" for those.

**Comment by elcorto** [comment]
See also #11 for additional info. They key is to make notebooks (i) CI-tested code and (ii) part of the built docs.
